frontend/src/common/backend.ts

```diff
@@ -355,6 +372,14 @@ class BillingService {
   }
 }
 
+class LicenseService {
+  public async getStatus(): Promise<LicenseStatusDto> {
+    return axiosAuth.get('/license/status').then(response => {
+      return new LicenseStatusDto(response.data.licensedSeats, response.data.usedSeats, new Date(response.data.expiresAt));
```

Comment:
`new Date(null)` is the Unix Epoch timestamp but should be `null` imo.

```suggestion
      return new LicenseStatusDto(response.data.licensedSeats, response.data.usedSeats, response.data.expiresAt ? new Date(response.data.expiresAt) : null);
```

Reply:
Fixed in f9c3f00f7f1d5197a054455571229cbe1ae3e716.